source/03_development/04_docker_images/02_ehrbase_db/02_use_image/002_using_docker_image.rst

```diff
@@ -0,0 +1,24 @@
+Run DB w/ default parameters
```

Comment:
Personally I don't like the w/ or w/o notation. This disturbs my reading ;-) 

May it is better to use the full words since we are a serious corporation ;-)